### ISSUE TITLE
[new feature] Image Slider offer

### ISSUE THREAD

**Issue by wndudqus** [dooboolab/dooboo-ui#103]
I offer an image slider.
Image sliders(also known as image carousels or slideshows) can be a convenient way to display multiple images.
I think that there are many cases where multiple images need to be displayed on mobile.
It will be useful in such a situation.

<img width="596" alt="스크린샷 2021-09-07 오후 11 55 15" src="https://user-images.githubusercontent.com/35381940/132366314-a775289e-17a0-44c6-9635-26f159d18bf2.png">

**Comment by yujong-lee** [comment]
Have you check `Pinch Zoom - image slider` in [demo page](https://dooboolab.github.io/dooboo-ui/)?
It is slightly different from your image, but similar.

**Comment by hyochan** [comment]
Yeah pinch zoom with carousel would be much beneficial. It'd be good if we can abstract the full feature as [ImageSlider].

**Comment by wndudqus** [comment]
I checked the pinch zoom slider and it was very similar to this feature. 
And I thought it might be good if the pinch zoom slider has a navigation button so that can user know how many picture are there

**Comment by wndudqus** [comment]
<img width="210" alt="스크린샷 2021-09-14 오후 10 49 15" src="https://user-images.githubusercontent.com/35381940/133269739-0ba4955d-50ea-45d5-9d29-f8119a38d00e.png">
such like this

**Comment by yujong-lee** [comment]
@wndudqus  I agree with your point!
